optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/plugin/websocket/PortableLocation.java

```diff
@@ -85,7 +85,7 @@ public boolean equals(Object o) {
         }
         PortableLocation that = (PortableLocation) o;
         return id == that.id &&
-                Objects.equals(description, that.description) &&
```

Comment:
May I ask what is the reason for removing the `Objects.equals`?

Reply:
Yes. Because I realized `description` can't be null, so I can safely do `descrption.equals(that.description)`.